website/docs/r/api_management_product_tag.html.markdown

```diff
@@ -0,0 +1,89 @@
+---
+subcategory: "API Management"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_product_tag"
+description: |-
+  Manages an API Management Product tag
+---
+
+# azurerm_api_management_product_tag
+
+Manages an API Management Product tag
+
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_api_management" "example" {
+  name                = "example-apim"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  publisher_name      = "My Company"
+  publisher_email     = "company@terraform.io"
+
+  sku_name = "Developer_1"
+}
+
+resource "azurerm_api_management_product" "example" {
+  product_id            = "test-product"
+  api_management_name   = azurerm_api_management.example.name
+  resource_group_name   = azurerm_resource_group.example.name
+  display_name          = "Test Product"
+  subscription_required = true
+  approval_required     = true
+  published             = true
+}
+
+resource "azurerm_api_management_tag" "example" {
+  api_management_id = data.azurerm_api_management.example.id
+  name              = "example-tag"
+}
+
+resource "azurerm_api_management_product_tag" "example" {
+  product_name        = azurerm_api_management_product.example.product_id
+  api_management_name = data.azurerm_api_management.example.name
+  resource_group_name = data.azurerm_resource_group.example.name
+  tag_name            = azurerm_api_management_tag.example1.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `api_management_name` - (Required) The name of the API Management Service. Changing this forces a new resource to be created.
+
+* `product_name` - (Required) The name of the API Management product. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group in which the API Management Service should be exist. Changing this forces a new resource to be created.
+
+* `tag_name` - (Required) The name which should be used for this API Management Tag. Changing this forces a new API Management Tag to be created.
```

Comment:
as the resource name is tag should this be
```suggestion
* `name` - (Required) The name which should be used for this API Management Tag. Changing this forces a new API Management Tag to be created.
```

so its `azurerm_api_management_product_tag.r.name` not `azurerm_api_management_product_tag.r.tag_name` ?